### ISSUE TITLE
Support Priority of DelayedJob.

### ISSUE THREAD

**Issue by joshuapinter** [jrgifford/delayed_paperclip#210]
Would you consider a PR to support the `priority` field in DelayedJob?

This would allow us to allow the priority of image/video processing to ensure more important jobs get done. 

Let me know what you think and if yes, I'll take a look.

Thanks!

And great gem by the way!

**Comment by jrgifford** [comment]
Of course! I know that I haven’t been super active recently but I’d be happy to bring that feature in. 

- James

**Comment by joshuapinter** [comment]
Excellent! 👍  

I think it'll be an important feature for our company so I'll try and get time approved for myself or somebody to work on it.
